Build failed on the windows

When I try to run on the windows, these errors happen.

  SEVERE: error[E0433]: failed to resolve: could not find `Resource` in `rlimit`
  SEVERE: error[E0425]: cannot find function `getrlimit` in crate `rlimit`

Exception: Build process failed.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,3 +1,7 @@
+## 0.0.2
+
+* Fixed the Windows build.
+
 ## 0.0.1
 
-* TODO: Describe initial release.
+* Initial release.

--- a/pubspec.yaml
+++ b/pubspec.yaml
@@ -1,6 +1,6 @@
 name: tor
 description: A multi-platform Flutter plugin for managing a Tor proxy. Based on arti.
-version: 0.0.1
+version: 0.0.2
 homepage: https://github.com/Foundation-Devices/tor
 
 platforms:

--- a/rust/src/lib.rs
+++ b/rust/src/lib.rs
@@ -15,7 +15,9 @@ use tor_rtcompat::tokio::TokioNativeTlsRuntime;
 use tor_rtcompat::BlockOn;
 
 pub use crate::error::tor_last_error_message;
+#[cfg(not(target_os = "windows"))]
 pub use crate::util::tor_get_nofile_limit;
+#[cfg(not(target_os = "windows"))]
 pub use crate::util::tor_set_nofile_limit;
 
 #[macro_use]

--- a/rust/src/util.rs
+++ b/rust/src/util.rs
@@ -4,12 +4,14 @@
 use crate::error::update_last_error;
 
 #[no_mangle]
+#[cfg(not(target_os = "windows"))]
 pub unsafe extern "C" fn tor_get_nofile_limit() -> u64 {
     let nofile_limit = unwrap_or_return!(rlimit::getrlimit(rlimit::Resource::NOFILE), 0);
     nofile_limit.0
 }
 
 #[no_mangle]
+#[cfg(not(target_os = "windows"))]
 pub unsafe extern "C" fn tor_set_nofile_limit(limit: u64) -> u64 {
     unwrap_or_return!(rlimit::increase_nofile_limit(limit), 0)
 }

